src/shared_modules/utils/filesystemHelper.h

```diff
@@ -21,6 +21,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <dirent.h>
+#include <vector>
```

Comment:
There's a reason to add this?

Reply:
Nope. Must have wanted to add it to some other file and got them mixed up. Removing this

Reply:
My bad @RamiroRD . This include was absolutely necessary. Dismiss this change request